src/pages/how-it-works.js

```diff
@@ -0,0 +1,309 @@
+import React from 'react';
+import { Button, Col, Row } from 'antd';
+import map from '../graphics/donation-map.png';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
This is interesting – let's keep track of what icon packages would be the best to use